Module 1 - What is a cluster.md

```diff
@@ -0,0 +1,61 @@
+
+Data of HPC cluster Examples:
+- Cedar: The communication backend is a high-speed low-latency 100 Gbit/s Intel Omni-Path fabric.
+- Graham: The communication backend is high-speed low-latency 56 Gbit/s (100 Gbit/s) Mellanox FDR (EDR) InfiniBand fabric.
+- Niagara: The communication backend is a high-speed low-latency 100 Gbit/s Mellanox EDR InfiniBand fabric leveraging leading edge Dragonfly+ topology. 
+- Béluga: The communication backend is a high-speed low-latency 100 Gbit/s Mellanox EDR InfiniBand fabric.
+### File systems
+[Lustre file system](https://docs.alliancecan.ca/wiki/Tuning_Lustre), known for its high performance in large-scale, complex computing environments; manages data distribution and access efficiently across multiple storage nodes. Stripe is a key of Lustre file system. For each file or directory, it is possible change the stripe size and stripe count parameters. Stripe size is the size of the smallest block of data that is allocated on the filesystem. Stripe count is the number of disks on which the data are spread.
+### Tiered Storage
+The tiering model for compute canada servers:
+1. Tier-0: node-local storage on fast SSDs. 
+2. Tier-1: shared project and home storage on a specific cluster (e.g. Cedar) which is split over SSDs and HDDs.
+3. Tier-2: storage across DRAC clusters accessed via globus over a dedicated Canada-wide network infrastructure for scientific HPC.
+4. Tier-3: users' personal devices.
+   
+[Data of HPC cluster Examples](https://alliancecan.ca/sites/default/files/2022-03/arc_current_state_report_0.pdf):
+- Graham: The tiered storage system ranges from small persistent ‘home’ (at over 130 TB total capacity), and persistent ‘project’ (at 16000TB), to high-speed non-persistent ‘scratch’ (at 3600 TB).
+- Cedar: The tiered storage system ranges from small persistent ‘home’ (at over 500 TB total
+capacity), and persistent ‘project’ (at 23 000 TB), to high-speed non-persistent ‘scratch’ (at
+5400 TB). 
+- Niagara: The tiered storage system ranges from small persistent ‘home’ (at 200 TB total capacity), and persistent ‘project’ (at 2000 TB), to high-speed nonpersistent ‘scratch’ (at 7000 TB), and very-high speed non-persistent ‘burst’ (at 230 TB).
+- Béluga: The tiered storage system ranges from small persistent ‘home’ (at over 100 TB total capacity), and persistent ‘project’ (at 25000 TB), to high-speed non-persistent ‘scratch’ (at 2600 TB).
+  
+![Tiering model. Photo by https://static.wixstatic.com](https://static.wixstatic.com/media/5446c2_002fa8b33249455b80a94926fc81cc26~mv2.jpg/v1/crop/x_0,y_118,w_1015,h_482/fill/w_1015,h_482,al_c,q_85,enc_auto/07032022_Tiering-Model.jpg)*Tiering model. Photo by https://static.wixstatic.com*
```

Comment:
I think I would go ahead and drop this image. The bullet points above are clear enough on their own.